_adr/80/index.adoc

```diff
@@ -20,3 +20,73 @@ permissioned groups.
 
 
 
+## Goals
+
+* Identify the preferred solution for the diagnostic gathering mechanism.
+* Provide guidelines for content that should be included.
+
+## Non-goals
+
+None
+
+## Stakeholders
+* Eng
+* BU
+* SRE
+
+## Current Architecture
+
+None
+
+## Proposed Architecture
+
+https://datatracker.ietf.org/doc/html/rfc2119[RFC-2119] keywords are used in this proposal.
+
+It is RECOMMENDED that managed services use https://source.redhat.com/groups/public/openshiftplatformsre/wiki/backplane_scripts[Back Plane]
+https://github.com/openshift/managed-scripts/[managed scripts] as the foundation of the  data gathering mechanism. Backplane has an 
+established authorization model that ensures that CEE have the permissions to run the scripts (KW: how does this work)?
```

Comment:
This [document](https://source.redhat.com/groups/public/openshiftplatformsre/wiki/backplane_cluster_permissions) describes how the permissions are set up.

RBAC rules are defined [here](https://github.com/openshift/managed-cluster-config/tree/master/deploy/backplane)

Reply:
I don't think it's necessary to include this information however. We should be able to assume that CEE and SRE can run any scripts provided.

Reply:
thanks, I linked the document you provided to help a reader understand the authz model, but as you suggest, I've included no details in this doc.